Assets/LeapMotionModules/Hands/Scripts/LeapHandsAutoRig.cs

```diff
@@ -0,0 +1,302 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Leap.Unity {
+  /**LeapHandAutoRig automates setting up the scripts that drive 3D skinned mesh hands. */
+  [ExecuteInEditMode]
```

Comment:
This attribute does nothing because the behaviour doesn't have an Awake or Update method.